requirements.txt

```diff
@@ -0,0 +1,5 @@
+cookiecutter>=1.4.0
+
+# Testing
+pytest==3.0.5
+pytest-cookies==0.2.0
```

Comment:
I don't think so, there are not a lot of requirements, I put this file just for documentation, and people using this template just will need to install `cookiecutter`